.buildkite/pipeline.yml

```diff
@@ -0,0 +1,52 @@
+steps:
+  - label: "build-gnu-x86-json"
+    commands:
+     - cargo build --release --features=json
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+      os: linux
+    plugins:
+      - docker#v3.8.0:
+          image: "rustvmm/dev:v12"
+          always-pull: true
+
+  - label: "build-musl-x86-json"
+    commands:
+     - cargo build --release --features=json --target x86_64-unknown-linux-musl
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+      os: linux
+    plugins:
+      - docker#v3.8.0:
+          image: "rustvmm/dev:v12"
+          always-pull: true
+
+  - label: "build-gnu-arm-json"
+    commands:
+     - cargo build --release --features=json
+    retry:
+      automatic: false
+    agents:
+      platform: arm.metal
+      os: linux
+    plugins:
+      - docker#v3.8.0:
+          image: "rustvmm/dev:v12"
+          always-pull: true
+
+  - label: "build-musl-arm-json"
+    commands:
+     - cargo build --release --features=json --target aarch64-unknown-linux-musl
```

Comment:
We should also add a `cargo check` with the json feature on both platforms.

Reply:
Yes, what is not checked is the reversed I think, that there are no warnings with no features.

Reply:
I believe this should be done by adding the `-D warnings` flag to the `cargo build` command, in the default rust-vmm-ci pipeline. WDYT?

This way we'd make this a default in rust-vmm, instead of having a custom step

Reply:
Sounds good to me, do you want to open a PR with your suggestion?

Reply:
yep, will do that

Reply:
See https://github.com/rust-vmm/rust-vmm-ci/pull/86